resources/test/fixtures/flake8_simplify/SIM300.py

```diff
@@ -13,3 +16,7 @@
 "yoda" == compare == 1
 "yoda" == compare == someothervar
 "yoda" == "yoda"
+age == YODA
+age < YODA
+age <= YODA
+
```

Comment:
Maybe add a test for two constants?
`THIS == THAT`

Reply:
Done in [b348a13](https://github.com/charliermarsh/ruff/pull/2255/commits/b348a133bd80d047aaebfca53ceae709a17b03b7)